src/catchErrors.js

```diff
@@ -0,0 +1,28 @@
+const { cases } = require('./cases');
+
+const checkURL = (text, toCase) => {
+  const errors = [];
+  const example = '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"';
+
+  if (!text) {
+    errors.push({
+      message: `Text to convert is required. Correct request is: ${example}.`,
+    });
+  }
+
+  if (!toCase) {
+    errors.push({
+      message: `"toCase" query param is required. Correct request is: ${example}.`,
+    });
+  }
+
+  if (!cases.includes(toCase) && toCase) {
+    errors.push({
+      message: `This case is not supported. Available cases: ${cases.join(', ')}.`,
+    });
+  }
+
+  return errors;
```

Comment:
If this function validates data, maybe makes sense to throw a new error if validation failed? This function can return nothing, BUT throw error if array of errors is not empty. This behaviour is more understandable, than array of errors.